lib/plugin-support/shared-worker-loader.js

```diff
@@ -15,8 +15,17 @@ class TestWorker {
 	constructor(id, file) {
 		this.id = id;
 		this.file = file;
+		this.sharingMap = new Map();
+		this.transferringMap = new Map();
 	}
 
+	sharing(...buffers) {
```

Comment:
@novemberborn Not sure if this is the API you have in mind.

Reply:
I think it has to return something new where you can share more, or publish.

Reply:
So like a new `TestWorker` initialized with the shared buffer(s)?

Reply:
Mmh, maybe a `Message` class